docs/builders/index.mdx

```diff
@@ -46,7 +46,6 @@ The OUTSCALE provider offers a flexible means of providing credentials for authe
 - Static credentials
 - Environment variables
 - Shared credentials file
-- Outscale Role
```

Comment:
but why ?

Reply:
We don't have this feature, we can't access credentials for Packer through Outscale Role